events_task/lib/my_app_web/templates/event/show.html.eex

```diff
@@ -0,0 +1,31 @@
+<h1>Show Event</h1>
+
+<ul>
+
+  <li>
+    <strong>Description:</strong>
+    <%= @event.description %>
+  </li>
+
+  <li>
+    <strong>Timestamp:</strong>
+    <%= @event.timestamp %>
+  </li>
+
+</ul>
+<span><%= link "Back", to: "/" %></span>
+<%= if @current_user do %>
+	<%= if @current_user.id == @event.author_id do %>
+	<span><%= link "Edit", to: event_path(@conn, :edit, @event) %></span>
+	<% end %>
+	<%= if Enum.member?(@user_email_list, @current_user.email) do %>
```

Comment:
Same as security issues, except not a security issue - but still, nothing stops user from creating two confirmations. In fact, you can even do it in browser - go to event show page, click confirm, go back using browser back button, click confirm, ...
This should be fixed using unique indexes like it was described for `user.email` - this is the only way to guarantee that user can't confirm same event twice.